common-security/README.md

```diff
@@ -0,0 +1,201 @@
+# Cask Common: Security
+
+**Introduction**
+
+Security framework that provides authentication and authorization.
+
+## Authorization
+
+The Authorization module contains an authorization system that can be used to control access to protected resources.
+
+### Definitions
+
+* Object: A resource that may be access-controlled (e.g. file, database table)
+* Subject: A user or group that acts on an object (e.g. you)
+* Permission: A type of action that a subject can do on an object (e.g. write)
+* ACL entry: Consists of an object type and unique identifier, subject type and unique identifier, and a permission
+
+### Usage
+
+To use authorization in your project, you must first run an `AuthorizationService`. This service
```

Comment:
Perhaps "you must run an...", as the service needs to be running always, correct?